composer.json

```diff
@@ -57,6 +58,7 @@
         "symfony/uid": "^7.0.3",
         "symfony/var-dumper": "^7.0.3",
         "tijsverkoyen/css-to-inline-styles": "^2.2.5",
+        "visus/cuid2": "^5.0.0",
```

Comment:
This maintainer isn't very active. I'd be very hesitant to base the framework on this package's support.